Find a functionally equivalent code:android.content.pm.Signature.areExactMatch:COMMENT
Method Modifier: public      static      hidden      
Comment:/**
 * Test if given {@link Signature} sets are exactly equal.
 *
 * @hide
 */

@@ -1,3 +1,3 @@
 {
-    return ArrayUtils.containsAll(a, b) && ArrayUtils.containsAll(b, a);
+    return (a.length == b.length) && ArrayUtils.containsAll(a, b) && ArrayUtils.containsAll(b, a);
 }

